API.Tests/Base/BaseTests.cs

```diff
@@ -5,98 +5,89 @@
-        {
-            if(accessToken != null)
+            if(token == null)
             {
-                return accessToken;
+                throw new Exception("NO JWT TOKEN!!!!!!!!");
             }
 
-            return await RenewAccessToken();
-        }
+            TestClient.DefaultRequestHeaders.Add("Authorization", "Bearer " + token);
 
-        private async Task<string> RenewAccessToken()
-        {
-            accessToken = await GetAccessToken();
-            return accessToken;
+            int id = (int)identityId;
+            TestClient.DefaultRequestHeaders.Add("IdentityId", id.ToString());
         }
 
-        private async Task<string> GetAccessToken()
+        private async Task<string> GetToken()
         {
-            Dictionary<string, string> dict = new Dictionary<string, string>();
-            dict.Add("client_id", "dex-api-client");
-            dict.Add("client_secret", "Q!P5kqCukQBe77cVk5dqWHqx#8FaC2fDN&bstyxrHtw%5R@3Cz*Z");
-            dict.Add("scope", "ProjectRead ProjectWrite UserRead UserWrite HighlightRead HighlightWrite");
-            dict.Add("grant_type", "client_credentials");
-
-            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, $"https://localhost:5004/connect/token")
+            if(accessToken != null)
             {
-                Content = new FormUrlEncodedContent(dict)
-            };
-
-            HttpClient client = new HttpClient();
-            HttpResponseMessage response = await client.SendAsync(request);
-            string token = JsonConvert.DeserializeObject<AccessTokenReponse>(await response.Content.ReadAsStringAsync()).Access_token;
+                return accessToken;
+            }
 
-            return token;
+            accessToken = await GetJwtAsync();
+            return accessToken;
         }
 
         protected async Task<string> GetJwtAsync()
         {
-            Dictionary<string, string> dict = new Dictionary<string, string>();
-            dict.Add("client_id", "dex-api-client");
-            dict.Add("client_secret", "Q!P5kqCukQBe77cVk5dqWHqx#8FaC2fDN&bstyxrHtw%5R@3Cz*Z");
-            dict.Add("scope", "ProjectRead ProjectWrite UserRead UserWrite HighlightRead HighlightWrite");
-            dict.Add("grant_type", "client_credentials");
-
-            HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, "https://localhost:5005/connect/token")
+            try
             {
-                Content = new FormUrlEncodedContent(dict)
-            };
-
-            HttpClient client = new HttpClient();
-
-            HttpResponseMessage response = await client.SendAsync(request);
-
-            string token = JsonConvert.DeserializeObject<AccessTokenReponse>(await response.Content.ReadAsStringAsync()).Access_token;
-
-            return token;
+                if(identityAddress == null)
+                {
+                    Console.WriteLine("IdentityAddress is null!!!!!!!!");
```

Comment:
A more professional message something like "No IdentityAddress" Or "IdentityAddress  is empty"